app/controllers/api_controller/renderer.rb

```diff
@@ -457,7 +457,7 @@ def fetch_typed_subcollection_actions(method, is_subcollection)
       return unless is_subcollection
       ctype = @req[:collection].to_sym
       sakey = "#{@req[:subcollection]}_subcollection_actions".to_sym
-      collection_config.fetch_path(ctype, sakey, method.to_sym)
+      collection_config[ctype][sakey] && collection_config[ctype][sakey][method.to_sym]
```

Comment:
Thanks @fryguy that sounds good, we'll go ahead with this PR, update more_core_extensions gem, then create another PR to leverage the new Gem + fetch_path. Nice thing with this approach is it will be available to all.